migrations/versions/edeebfad2670_add_q4_2023_eia860m_data.py

```diff
@@ -0,0 +1,28 @@
+"""Add Q4 2023 EIA860m data
```

Comment:
Was this revision necessary? We aren't changing the table schema at all.

Reply:
Maybe not, I think I got an alembic error, but maybe I should have just run alembic upgrade head. I'll try that.